src/target/at32f43x.c

```diff
@@ -89,7 +89,14 @@ static bool at32f43_mass_erase(target_s *target);
 #define AT32F43x_2K_OB_COUNT 256U
 #define AT32F43x_4K_OB_COUNT 2048U
 
-#define DBGMCU_IDCODE 0xe0042000U
+#define AT32F43x_DEBUG            0xe0042000U
+#define DBGMCU_IDCODE             (AT32F43x_DEBUG + 0U)
+#define AT32F43x_DEBUG_CTRL       (AT32F43x_DEBUG + 4U)
+#define AT32F43x_DEBUG_APB1_PAUSE (AT32F43x_DEBUG + 8U)
+
+#define AT32F43x_DEBUG_CTRL_SLEEP      0x7U
```

Comment:
For consistency with other implementations like the [STM32H5 implementation](https://github.com/blackmagic-debug/blackmagic/blob/a4934aed4fdf61f764c93b2065111e34c0cf603e/src/target/stm32h5.c#L101-L117) that's coming in an overhaul of the watchdogs situation, please express this as the 3 different sleep control bits (so they have names) and them `|` them together below.

Reply:
Detailed as three control bits with names from refman.